qa/sc_evm_closed_forger.py

```diff
@@ -63,26 +80,86 @@ def tryMakeForgerStake(self, sc_node, owner_address, blockSignPubKey, vrf_public
         # status == 1 is succesful
         return status == 1
 
+
+    def tryOpenStakeForgerList(self, sc_node, forgerIndex, api_error_expected=False, forger_node=None):
+        j = {
+            "forgerIndex": forgerIndex,
+        }
+        request = json.dumps(j)
+        jsonRes = sc_node.transaction_openStakeForgerList(request)
+        logging.info(json.dumps(jsonRes))
+        self.sc_sync_all()
+
+        if api_error_expected:
+          assert_true("error" in jsonRes)
+          return False
+        else:
+          assert_true("result" in jsonRes)
+
+        if forger_node is None:
+            generate_next_block(sc_node, "first node")
+        else:
+            generate_next_block(forger_node, "first node")
+        self.sc_sync_all()
+
+        # check we had a failure in the receipt
+        tx_hash = jsonRes['result']["transactionId"]
+        logging.info("Getting receipt for txhash={}".format(tx_hash))
+
+        receipt = sc_node.rpc_eth_getTransactionReceipt(add_0x_prefix(tx_hash))
+        status = int(receipt['result']['status'], 16)
+        # status == 1 is succesful
+        return status == 1
+
     def run_test(self):
         ft_amount_in_zen = Decimal('500.0')
         ft_amount_in_wei = convertZenToWei(ft_amount_in_zen)
 
         self.sc_ac_setup(ft_amount_in_zen=ft_amount_in_zen)
 
         sc_node_1 = self.sc_nodes[0]
+        sc_node_2 = self.sc_nodes[1]
+        mc_node = self.nodes[0]
 
         evm_address_sc_node_1 = remove_0x_prefix(self.evm_address)
 
         stakeList = sc_node_1.transaction_allForgingStakes()["result"]['stakes']
         assert_equal(1, len(stakeList))
 
+        # check we have the expected content of the close forger list
+        allowedForgerList = sc_node_1.transaction_allowedForgerList()["result"]
+        assert_equal(len(allowedForgerList['allowedForgers']), 3)
```

Comment:
Expected and actual parameters are switched. Also in other assert_equal calls below

Reply:
Done